app/models/audit_activity/investigation/update_risk_level_validation.rb

```diff
@@ -0,0 +1,29 @@
+class AuditActivity::Investigation::UpdateRiskLevelValidation < AuditActivity::Investigation::Base
+  def self.from(*)
+    raise "Deprecated - use UpdateRiskLevelValidation.call instead"
+  end
+
+  def self.build_metadata(investigation, rationale)
+    updated_values = investigation.previous_changes.slice(:risk_validated_at).merge(investigation.previous_changes.slice(:risk_validated_by))
+
+    {
+      updates: updated_values,
+      rationale: sanitize_text(rationale)
+    }
+  end
+
+  def title(*)
+    if metadata["updates"]["risk_validated_at"].second
```

Comment:
```suggestion
    if metadata.dig("updates", "risk_validated_at", 1)
```
not a big deal but this would avoid raising and error if on of the keys does not exist